File: Python/Iniciante/1019 - Conversão de Tempo.py
"""
Autor: Pierre Vieira
Data da submissão: 19/01/2019 21:40:31
"""
n_segundos = int(input())
horas = n_segundos//3600
minutos = (n_segundos%3600)//60
segundos = ((n_segundos%3600)%60)
print("{}:{}:{}".format(horas, minutos, segundos))

File: C#/Iniciante/1019 - Conversão de Tempo.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 14:19:56
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int horas, minutos, segundos;
            segundos = int.Parse(Console.ReadLine());
            horas = segundos / 3600;
            segundos %= 3600;
            minutos = segundos / 60;
            segundos %= 60;
            Console.WriteLine(horas+":"+minutos+":"+segundos);
        }
    }
}
